netlify-static/2020/index.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
+      integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T"
+      crossorigin="anonymous"
+    />
+    <title>PyOhio 2020 - Save The Date!</title>
+    <meta
+      name="description"
+      content="PyOhio is a FREE annual Python conference. July 25-26, 2020 in Columbus, OH."
+    />
+    <meta name="twitter:card" content="summary_large_image" />
+    <meta
+      name="twitter:description"
+      content="PyOhio is a FREE annual Python conference. July 25-26, 2020 in Columbus, OH."
+    />
+    <meta
+      name="twitter:image"
+      content="https://www.pyohio.org/2020/pyohio.jpg"
+    />
+    <meta name="twitter:site" content="@pyohio" />
+    <meta name="twitter:title" content="PyOhio 2020" />
+    <meta
+      property="og:image"
+      content="https://www.pyohio.org/2020/pyohio.jpg"
+    />
+    <meta property="og:title" content="PyOhio 2020" />
+  </head>
+  <body>
+    <div class="jumbotron">
+      <h1>Save the date!</h1>
+      <img src="pyohio_600.png" />
```

Comment:
```suggestion
      <img src="pyohio_600.png" style="width: 100%;" />
```
I looked on my phone and the logo was way too big, this will fix it